ci-operator/jobs/red-hat-storage/lvm-operator/red-hat-storage-lvm-operator-main-presubmits.yaml

```diff
@@ -49,3 +49,77 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )unit-test,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^main$
+    - ^main-
+    cluster: build03
+    context: ci/prow/e2e-test
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/cloud: aws
+      ci-operator.openshift.io/cloud-cluster-profile: aws
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-red-hat-storage-odf-multicluster-orchestrator-main-e2e-test
```

Comment:
This name needs to be fixed to used lvm operator.

Reply:
Updated